File: Iniciante/1099 - Soma de Ímpares Consecutivos II/Python/1099.py
def main():
    N = int(input())
    soma_impares(N)


def soma_impares(n):
    contador = 0
    while contador < n:
        X, Y = input().split()
        X = int(X)
        Y = int(Y)
        maior = 0
        menor = 0
        soma = 0

        if X > Y:
            maior = X
            menor = Y
        else:
            maior = Y
            menor = X

        somador = maior - 1

        while somador > menor:
            if somador % 2 != 0:
                soma += somador
            somador -= 1

        contador += 1
        print(soma)


main()

File: Iniciante/1099 - Soma de Ímpares Consecutivos II/JavaScript/1099.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  somaImpares(lines);
}

function somaImpares(l) {
  let N = parseInt(l.shift());
  for (let i = 0; i < N; i++) {
    let entradasAtual = l[i].split(" ");
    let X = parseInt(entradasAtual[0]);
    let Y = parseInt(entradasAtual[1]);
    let maior = 0;
    let menor = 0;
    let soma = 0;

    if (X > Y) {
      maior = X;
      menor = Y;
    } else {
      maior = Y;
      menor = X;
    }
    let somador = maior - 1;

    while (somador > menor) {
      if (somador % 2 !== 0) {
        soma += somador;
      }
      somador -= 1;
    }
    console.log(soma);
  }
}

main();
